frontend/src/main/pages/operator/components/ProfileView.tsx

```diff
@@ -1,12 +1,167 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useSelector } from '../../../../state/utilities/use_selector'
+
+import { Typography, IconButton, Avatar, Paper, Grid, styled, Button, TextField, Dialog, DialogActions, DialogContent, DialogContentText, DialogTitle } from '@mui/material';
+import { Edit } from '@mui/icons-material';
+
+const roleEnums = {
```

Comment:
I'd prefer to see this come from the backend (either statically or in a code table)

Reply:
added this to the /me route and doing the string conversion / enumeration from there